docs/network/quickstart/qiuck-start.md

```diff
@@ -0,0 +1,215 @@
+
+// Initialize the logstore client
+const logStoreClient = new LogStoreClient({
+  auth: {
+    privateKey,
+  },
+ });
+
+//get the stream
+const stream = await logStoreClient.getStream(streamId);
+console.log({ stream });
+
+//stake to the stream
+const stakeAmount = 10;
+const response = await logStoreClient.stakeOrCreateStore(
+   stream.id,
+   stakeAmount
+);
+```
+
+**Smart Contracts:**
+
+By calling the `stake` method of the store manager contract, we can directly stake some LSAN tokens into the smart contract.
+
+```jsx
+// StoreManager.sol
+function stake(string memory streamId, uint amount)
+```
+
+### Publishing to Stream
+
+**Client:**
+
+```jsx
+import { LogStoreClient, CONFIG_TEST } from "@logsn/client";
+
+async function publishMessage() {
+  // Initialize the logstore client
+  const logStoreClient = new LogStoreClient({
+    auth: {
+      privateKey,
+    },
+  });
+  const stream = await logStoreClient.getStream(streamId);
+  console.log({ stream });
+
+  const stakeAmount = 10;
+  const response = await logStoreClient.stakeOrCreateStore(
+    stream.id,
+    stakeAmount
+  );
+  console.log({ response });
+
+  const message = {
+    type: "client:publish",
+    ts: Date.now(),
+  };
+
+  await logStoreClient.publish(stream, message);
+  console.log("Sent successfully: ", message);
+}
+
+await publishMessage();
+```
+
+### **Demo**
+
+Coming soon!
+
+## Querying from a stream
+
+Before we can query for a stream, we need to stake some funds using our wallet in order to authenticate us
+
+### Staking for querying
+
+Using the following methods, one can stake some funds to a stream.
+
+**Cli:** 
+
+`logstore query stake --host [https://polygon-rpc.com](https://polygon-rpc.com/) --wallet ${walletAddress} ${amount}`
+
+**Client:** 
+
+```
+import { LogStoreClient, CONFIG_TEST } from "@logsn/client";
+
+// Initialize the logstore client
+const logStoreClient = new LogStoreClient({
+  auth: {
+    privateKey,
+  },
+ });
+
+//stake to the stream
+const stakeAmount = 10;
+const response = await logStoreClient.queryStake(
+   stakeAmount
+);
+```
+
+**Smart Contracts:**
+
+By calling the `stake` method of the query manager contract, we can directly stake some LSAN tokens into the smart contract.
+
+```jsx
+// QueryManager.sol
+function stake(uint amount) public
+```
+
+### **Querying the stream**
+
+```jsx
+import { LogStoreClient, CONFIG_TEST } from "@logsn/client";
+
+const stream = await logStoreClient.getStream(streamId);
+// Initialize the logstore client
+const logStoreClient = new LogStoreClient({
+  auth: {
+    privateKey,
+  },
+ });
+
+const query = await client.query(
+    stream.id,
+    {
+      // should see the recently send messages, along with 3 identical ones from storage
+      last: 6,
+    },
+    (message) => {
+      // Do something with the messages as they are received
+      console.log(JSON.stringify(message));
+    }
+  );
+```
+
+### **Demo**
+
+Coming soon!z
```

Comment:
What is this?

Reply:
i was going to include some video demos,  but the network was not going through coupled with the upgrade of the network so i just put that there to come back to it. ot i should take it out completely?